### ISSUE TITLE
File save/load not possible 

### ISSUE THREAD

**Issue by swhite2401** [atcollab/at#340]
Several issues experienced with saving and loading files:

-Sometimes the first element (ringparam) is initialized with FamName='', this prevents the lattice from loading
properly. This is consecutive to a save_mat()
-When the Particle attribute of this first element is not a particle object and does not have the proper attributes
loading also fails

I add @simoneliuzzo in this thread because he found the problem generating a lattice from scratch (I think?), he can provide more information if useful

**Comment by lfarv** [comment]
- Is it specific to .mat files ?
- May I have an example of wrong file, with the way it has been generated (from Matlab of python ?). For instance, why the Particle attribute of a RingParam would not be a particle ?

**Comment by swhite2401** [comment]
```
import at

q = at.Quadrupole('QUAD', 0)
ring = at.Lattice([q],energy=1,periodicity=1)

at.save_lattice(ring, './quad.mat')
ring = at.load_lattice('quad.mat')
```

This code makes it fails, if I force FamName of RingParam element to a value it seems to work (for the first issue)

**Comment by lfarv** [comment]
Fixed in #341
